contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2024-03-21
+title: 'PostHog is now up to 12x cheaper'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* It is now possible to use PostHog's product and web analytics for up to ~12x less, depending on your volumes
+* This change applies to all existing and new customers for any events that have no person properties set
+* The average customer can save X%
+* We are intending on more price cuts, this is the start!
+
+## Pricing before and after
+
+| Tier (up to) | Personless Price (per event) | Previous pricing / our new pricing for events with person propertie |
+| ---- | ----- | ----- | ----- |
+| 1M | $0 | $0 | $0 |
+| 2M | $0.00002 |  $0.000248 |
+| 15M | $0.00002 |  $0.000104 |
+| 50M | $0.00002 |  $0.0000655 |
+| 100M | $0.00002 |  $0.0000364 |
+| 250M | $0.000009 *price changes here | $0.0000187 |
+| ∞ | $0.000009 | $0.00001 |
+
+(we can offer further discounts if you are beyond 250M events/month regardless - at that scale, we suggest you talk to us)
+
+## Why we're excited to be the cheapest
+
+* We think this makes us the best value product analytics software by a long way
+* Everyone else has far fewer features available in their free or lower priced tiers - we only limit team-oriented features (like advanced permissioning). Instead of this, by changing slightly the data we capture, we can pass these savings along to customers directly.
+* We have lots of other products, we think we're better off long term making a little money from a lot of products.
+
+<INSERT GRAPH COMPARING US TO COMPETITORS>
+
+## What's the downside of sending this type of event?
+
+If you are a price sensitive customer, we'd recommend you make this change for anonymous traffic.
```

Comment:
Personally, I don't think we should recommend this so strongly. We instead need to foreground that there are downsides to using this approach and couch the explanation in the fact that users have greater control. 

Basically, if this is positioned as a way to save money, I can see users turning it on and then asking us why data is looking weird or things aren't working.